lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -901,7 +901,7 @@ def commit(order, message_type, retry_logic = nil, trace_number = nil)
             request.call(remote_url(:secondary))
           end
 
-        authorization = order.include?('<TokenTxnType>GT</TokenTxnType>') ? response[:safetech_token] : authorization_string(response[:tx_ref_num], response[:order_id])
+        authorization = order.include?('<TokenTxnType>GT</TokenTxnType>') ? authorization_string(response[:safetech_token], response[:card_brand]) : authorization_string(response[:tx_ref_num], response[:order_id])
```

Comment:
I think we need to rethink this `authorization` string we are building. There is a loose pattern for these things that we follow, which is not well documented, but it helps standardize these strings at the individual gateway level.

TLDR; We don't want to conditionally build the string to be different, depending on the transaction. Regardless of the values returned in the response from the gateway, we want to maintain a consistent structure.

Example:

```
authorization = authorization_string(r[field1], r[field2], r[field3])

# authorization = 'val1;val2;val3'
# authorization = 'val1;;val3'
# authorization = 'val1;;'
```

This way, wherever we split this string, we can reliably retrieve the index of the authorization and _know_ that it should be value we expect.

For this, we should take the existing `authorization_string` call and append the values we would like to include in this string, so that the length of this string (after it is split into the individual values) is always the same.